packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/package.json

```diff
@@ -0,0 +1,127 @@
+{
+  "name": "@<%=projectName%>/<%=projectPackageName%>",
+  "version": "0.0.0",
+  "description": "<%=projectPackageName%> - SDK",
+  "source": "src/index.ts",
+  "main": "cjs/index.js",
+  "module": "esm2020/index.js",
+  "node": "cjs/index.js",
+  "es2020": "esm2020/index.js",
+  "esm2020": "esm2020/index.js",
+  "es2015": "esm2015/index.js",
+  "esm2015": "esm2015/index.js",
+  "types": "index.d.ts",
+  "sideEffects": false,
+  "exports": {
+    "./package.json": {
+      "default": "./package.json"
+    },
+    ".": {
+      "main": "./cjs/index.js",
+      "module": "./esm2020/index.js",
+      "es2020": "./esm2020/index.js",
+      "esm2020": "./esm2020/index.js",
+      "node": "./cjs/index.js",
+      "default": "./esm2020/index.js",
+      "typings": "./index.d.ts",
+      "import": "./esm2020/index.js",
+      "require": "./esm2020/index.js"
+    }
+  },
+  "scripts": {
+    "clean": "rimraf test/ test-dev/ dist/ dist-dev/ dist-test/ build/",
+    "lint:ci": "eslint '**/*[jt]s' --quiet --format junit --output-file ./dist-lint/result.xml",
+    "lint": "eslint '**/*[jt]s' --cache",
+    "start": "tsc-watch -b tsconfigs/esm2020 --noClear --onFirstSuccess \"yarn files:pack --watch\"",
+    "build": "yarn build:cjs && yarn build:esm2015 && yarn build:esm2020 && yarn files:pack",
+    "build:cjs": "swc src -d dist/cjs -C module.type=commonjs -q",
+    "build:esm2015": "swc src -d dist/esm2015 -C module.type=es6 -q",
+    "build:esm2020": "tsc -b tsconfigs/esm2020",
+    "set:version": "set-version --placeholder 0.0.0",
+    "resolve": "node -e 'process.stdout.write(require.resolve(process.argv[1]));'",
+    "generate": "schematics @ama-sdk/schematics:typescript-core --swagger-spec-path ./swagger-spec.yaml",
+    "swagger:regen": "schematics @ama-sdk/schematics:typescript-core --swagger-spec-path ./swagger-spec.yaml && yarn clear-index",
+    "clear-index": "node scripts/clear-index.js",
+    "files:pack": "node scripts/files-pack.js",
+    "test": "jest --passWithNoTests",
+    "publish:package": "npm publish ./dist",
+    "postinstall": "husky install",
+    "generate:mock": "schematics @ama-sdk/schematics:mock",
+    "doc:generate": "node scripts/override-readme.js && typedoc && node scripts/restore-readme.js",
+    "tools:changelog": "standard-version"
+  },
+  "lint-staged": {
+    "**/*.(js|ts)": [
+      "yarn eslint --quiet --fix --global fast"
+    ]
+  },
+  "dependencies": {
+    "@o3r/dev-tools": "~<%= sdkCoreVersion %>",
+    "@swc/helpers": "^0.5.0",
+    "crc-32": "~1.2.0",
+    "tslib": "^2.4.1",
+    "uuid": "^8.3.2"
+  },
+  "peerDependenciesMeta": {
+    "isomorphic-fetch": {
+      "optional": true
+    }
+  },
+  "devDependencies": {
+    "@angular-devkit/core": "~15.2.0",
```

Comment:
The versions may be aligned on the workspace one to avoid maintaining this file
(same for the other deps)

Reply:
already the case (at least for angular ones)